.github/workflows/reusable-testing.yml

```diff
@@ -39,6 +39,17 @@ jobs:
         with:
           files: "composer.json, phpunit.xml.dist"
 
+      - name: Get composer.json PHP version
+        id: parse-composer
+        run: |
+          echo "COMPOSER_PHP_VERSION=$(jq '.require.php' composer.json | sed 's/[^0-9.|]//g' | jq -Rc 'split("|") | min | tonumber')" >> $GITHUB_OUTPUT;
```

Comment:
One thought I had was that we should use `composer show` or `composer check-platform-reqs` to have composer reconcile the minimum PHP version from all of the dependencies.

_However_, we could end up in an (unlikely) scenario where the minimum PHP version is sufficiently high, causes all of the tests to be skipped, and the entire build passes without running any tests.

I kinda feel like we should read a `MINIMUM_TEST_PHP_VERSION` variable from a `.env` file so it's:

1. Explicitly defined.
2. Entirely outside of Composer.

Reply:
@schlessera @swissspidy Do either of you have a preference?

Reply:
I guess I'm open to using the `require.php` in `composer.json`. We can treat that as the explicit "minimum PHP version".